src/Files.App/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs

```diff
@@ -24,7 +24,9 @@ public HorizontalMultitaskingControl()
             InitializeComponent();
             tabHoverTimer.Interval = TimeSpan.FromMilliseconds(500);
             tabHoverTimer.Tick += TabHoverSelected;
-        }
+
+			RightPaddingColumn.Width = new GridLength(App.GetAppWindow(App.Window).TitleBar.RightInset);
```

Comment:
I think we need to test this with RTL languages. For RTL it could be LeftInset instead of RightInset
https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.windowing.appwindowtitlebar.leftinset?view=windows-app-sdk-1.1

Reply:
Isn't the width the same?

Reply:
I understand for RTL RightInset will be 0

Reply:
That's a good point